vector/src/main/java/im/vector/app/core/ui/views/SendStateImageView.kt

```diff
@@ -38,7 +38,7 @@ class SendStateImageView @JvmOverloads constructor(
     }
 
     fun render(sendState: SendStateDecoration) {
-        isVisible = when (sendState) {
+        isInvisible = !when (sendState) {
```

Comment:
Nit: Technically `!when` is fine of course, but not common, so for code clarity I would change it to `when` and replace `true` to `false` and `false` to `true` in each cases below.

Reply:
good point :+1:  https://github.com/vector-im/element-android/pull/4610/commits/3ff3507fa1a9627e13a39a6ce5edfb4e572470e5

Reply:
updated https://github.com/vector-im/element-android/pull/4610/commits/9ed72fd1eb37b4e5df0489f3de8c038c34e49ede

and with all the extra paddings removed https://github.com/vector-im/element-android/pull/4610/commits/23bc867b95d7f96e3dc551a2a5374bcb7ba1cfb7

| BEFORE | AFTER |
| --- | --- |
|![Screenshot_20211207_134956](https://user-images.githubusercontent.com/1848238/145040998-b69cee20-8304-49e5-a4a8-eed0ea34b193.png)|![Screenshot_20211207_134837](https://user-images.githubusercontent.com/1848238/145040789-3fb4cc9a-d7b2-4f97-8a47-60fd7af209d1.png)

currently checking with design if the other message types should extend to the gutter

Reply:
have sync with design, going with a consistent 8dp padding 

| 0dp | 8dp |
| --- | --- |
|![0-padding](https://user-images.githubusercontent.com/1848238/145055517-8e76bc9d-3a41-4bd5-bd09-1fd1cafa0284.png)|![8-padding](https://user-images.githubusercontent.com/1848238/145055514-3a47147d-5b2c-4d14-a80f-df93c4bc8189.png)